src/proof_system/preprocess.rs

```diff
@@ -168,6 +177,23 @@ impl TurboComposer {
             domain_4n.coset_fft(&selectors.fourth_sigma),
             domain_4n,
         );
+
+        let table_1_eval_4n = Evaluations::from_vec_and_domain(
```

Comment:
Its unclear how can we ever use an arity 3 table (previously declared) if we hardcode the table size to `4`